build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -49,6 +50,41 @@ public static BuildTargetResult GenerateVersionBadge(BuildTargetContext c)
         }
         
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows)]
+        public static BuildTargetResult CopyNativeSymbols(BuildTargetContext c)
+        {
+            var nativeSymbolsRoot = Path.Combine(Dirs.Output, "obj", "nativeSymbols");
+            var nativeSymbolsZipFileDir = Path.Combine(Dirs.Intermediate, "nativeSymbols");
+
+            Utils.DeleteDirectory(nativeSymbolsRoot);
+            Directory.CreateDirectory(nativeSymbolsRoot);
+
+            Utils.DeleteDirectory(nativeSymbolsZipFileDir);
+            Directory.CreateDirectory(nativeSymbolsZipFileDir);
+
+            foreach (var file in Directory.GetFiles(Dirs.CorehostLatest, "*.pdb", SearchOption.TopDirectoryOnly))
```

Comment:
Should we be publishing CoreHostLatest here or CoreHostLocked?

As we up the build number and the host is locked on some previous build, the pdbs in this package won't be representative of the binaries we shipped if we don't use CoreHostLocked.